### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: src/Templating/Helper/BlockHelper.php [hunk 1]
```diff
@@ -341,7 +341,10 @@ protected function computeAssets(BlockContextInterface $blockContext, array &$st
         ];
     }
 
-    private function startTracing(BlockInterface $block): array
+    /**
+     * @internal since sonata-project/block-bundle 3.16
+     */
+    protected function startTracing(BlockInterface $block): array
```
Comment (phansys) [inline src/Templating/Helper/BlockHelper.php]: The visibility should be private.

File: src/Templating/Helper/BlockHelper.php [hunk 2]
```diff
@@ -341,39 +341,10 @@ protected function computeAssets(BlockContextInterface $blockContext, array &$st
...
-                'age' => 0,
-            ],
-            'assets' => [
-                'js' => [],
-                'css' => [],
-            ],
-        ];
-    }
-
-    private function stopTracing(BlockInterface $block, array $stats): void
```
Comment (phansys) [inline src/Templating/Helper/BlockHelper.php]: All the methods that were marked with `@internal` at 3.x should remain private in 4.x.

Comment (greg0ire) [reply src/Templating/Helper/BlockHelper.php]: Oh ok sorry

File: src/Templating/Helper/BlockHelper.php [hunk 3]
```diff
@@ -341,38 +341,9 @@ protected function computeAssets(BlockContextInterface $blockContext, array &$st
...
-            ],
-            'assets' => [
-                'js' => [],
-                'css' => [],
-            ],
-        ];
-    }
-
+    /**
+     * @internal since sonata-project/block-bundle 3.16
```
Comment (phansys) [inline src/Templating/Helper/BlockHelper.php]: If these `@internal` annotations (which were already resolved using private visibility in 4.x) are intended to be removed in another PR, I think this one is ready.

Comment (greg0ire) [reply src/Templating/Helper/BlockHelper.php]: Yeah that's for the cleanup PR I guess #602
